pincer/objects/guild/guild.py

```diff
@@ -474,19 +477,326 @@ async def modify_member(
             Whether the member is deafened |default| :data:`None`
         channel_id : Optional[:class:`~pincer.utils.snowflake.Snowflake]
             Voice channel id to move to |default| :data:`None`
+        reason : Optional[:class:`str`]
+            audit log reason |default| :data:`None`
+            
         Returns
         -------
         :class:`~pincer.objects.guild.member.GuildMember`
             The new member object.
         """
         ...
 
-    async def modify_member(self, _id: int, **kwargs) -> GuildMember:
+    async def modify_member(self, _id: int, reason=None, **kwargs) -> GuildMember:
         data = await self._http.patch(
-            f"guilds/{self.id}/members/{_id}", data=kwargs
+            f"guilds/{self.id}/members/{_id}",
+            data=kwargs,
+            headers=remove_none({"X-Audit-Log-Reason":reason))
+        )
+        return GuildMember.from_dict(construct_client_dict(self._client, data))
+
+      
+    @overload
+    async def create_channel(
+        self, 
+        name: str,               
+        type: Optional[ChannelType] = None,
+        topic: Optional[str] = None,
+        bitrate: Optional[int] = None,
+        user_limit: Optional[int] = None,
+        rate_limit_per_user: Optional[int] = None,
+        position: Optional[int] = None,
+        permission_overwrites: Optional[List[Overwrite]] = None,
+        parent_id: Optional[Snowflake] = None,
+        nsfw: Optional[bool] = None
+    ) -> Channel:
```

Comment:
You have a big chunk of random whitespace there (mentioning it since it doesn't seem to show up on the diff below)
```suggestion
    async def create_channel(
        self,
        *,
        name: str,
        type: Optional[ChannelType] = None,
        topic: Optional[str] = None,
        bitrate: Optional[int] = None,
        user_limit: Optional[int] = None,
        rate_limit_per_user: Optional[int] = None,
        position: Optional[int] = None,
        permission_overwrites: Optional[List[Overwrite]] = None,
        parent_id: Optional[Snowflake] = None,
        nsfw: Optional[bool] = None
    ) -> Channel:
```